src/Umbraco.Web.UI.Client/src/views/components/editor/umb-editor-content-header.html

```diff
@@ -16,7 +16,7 @@
                     <p tabindex="0" class="sr-only">{{a11yMessage}}</p>
                 </div>
                 <div class="umb-editor-header__name-wrapper">
-                    <label for="headerName" class="sr-only">{{a11yName}}</label>
+                    <label for="headerName" class="sr-only" aria-label="{{a11yName}}">{{a11yName}}</label>
```

Comment:
`aria-label` is not needed here since there is a `<label>` element being used already :-)